tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -267,6 +268,38 @@ def _run(self):
                                     rebal_files=volume.rebal_files,
                                     rebal_data=volume.rebal_data
                                 ).save()
+                    connection_count = None
+                    connection_active  = None
+                    # Product: Community          Capacity:  25.00 GiB(raw bricks)
+                    # Status: HEALTHY                        3.00 GiB(raw used)
+                    # Glusterfs: 3.9.0                         87.00 GiB(usable from volumes)
+                    # OverCommit: Yes               Snapshots:   0
+                    # Nodes       :  2/  2		  Volumes:   4 Up
+                    # Self Heal   :  0/  0		             0 Up(Degraded)
+                    # Bricks      :  7/  7		             0 Up(Partial)
+                    # Connections :  1/   2                     0 Down
+                    cmd = cmd_utils.Command('gstatus -s')
+                    out, err, rc = cmd.run()
+                    if not err:
+                        # Connections :  0/   0        1 Down
+                        p = subprocess.Popen(
```

Comment:
i have converted cmd = cmd_utils.Command('gstatus -s') but for grep i have used subprocess.
cmd = cmd_utils.Command('gstatus -s | grep Connection') is not possible because ansible command module does not support pip symbol.